monkey/tests/unit_tests/monkey_island/cc/resources/auth/test_register.py

```diff
@@ -49,6 +43,18 @@ def test_register_successful(monkeypatch, make_registration_request, mock_authen
     mock_authentication_service.reset_repository_encryptor.assert_called_once()
 
 
+def test_register_string_response(
```

Comment:
Make this test parameterized and assert all kinds of invalid requests (at least when we get jinja template, when we get errors)